### PR TITLE
Job conf updates, sync with earlier tutos.

### PR DESCRIPTION
**Pull request by hexylena**

### CODE REVIEW

File: topics/admin/tutorials/connect-to-compute-cluster/tutorial.md [hunk 1]
```diff
@@ -81,31 +77,50 @@ be taken into consideration when choosing where to run jobs and what parameters
...
+>       roles:
+>    +    - galaxyproject.repos
+>    +    - galaxyproject.slurm
+>         - galaxyproject.postgresql
+>         - role: natefoo.postgresql_objects
+>           become: true
+>    ```
 >
-> 4. Add the slurm variables to your  `group_vars/galaxyservers.yml`:
+> 4. Add the slurm ariables to your `group_vars/galaxyservers.yml`:
```
Comment (nsoranzo) [inline topics/admin/tutorials/connect-to-compute-cluster/tutorial.md:96]:
```suggestion
> 4. Add the slurm variables to your `group_vars/galaxyservers.yml`:
```

**Review by Slugger70 — DISMISSED** [review]

File: topics/admin/tutorials/job-destinations/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,641 @@
...
+> 2. As usual, we need to instruct Galaxy of where to find this file:
+>
+>    ```diff
+>    --- a/group_vars/galaxyservers.yml
+>    +++ b/group_vars/galaxyservers.yml
+>    @@ -81,6 +81,7 @@ galaxy_local_tools:
+>     - testing.xml
+>     galaxy_dynamic_job_rules:
+>       - my_rules.py
+>    +  - map_resources.py
```
Comment (natefoo) [inline topics/admin/tutorials/job-destinations/tutorial.md:570]: Ditto indentation here

File: topics/admin/tutorials/pulsar/tutorial.md [hunk 3]
```diff
@@ -367,7 +410,8 @@ Some of the other options we will be using are:
 >
 >    {% raw %}
 >    ```yaml
->    galaxy_server_url: #please put the your Galaxy server's fqdn or ip address here (or the fqdn or ip address of the RabbitMQ server).
+>    galaxy_server_url: # Important!!!
+>    # Put the your Galaxy server's FQDN (or the FQDN of the RabbitMQ server).
```
Comment (natefoo) [inline topics/admin/tutorials/pulsar/tutorial.md:414]:
```suggestion
>    # Put your Galaxy server's fully qualified domain name (FQDN) (or the FQDN of the RabbitMQ server) above.
```

**Review by natefoo — COMMENTED** [review]
Amazing work as always @hexylena. Thanks!!

File: topics/admin/tutorials/connect-to-compute-cluster/slides.html [hunk 4]
```diff
@@ -106,10 +114,8 @@
```
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/admin/tutorials/connect-to-compute-cluster/slides.html:114]: done.

File: topics/admin/tutorials/connect-to-compute-cluster/slides.html [hunk 5]
```diff
@@ -238,6 +256,12 @@
 
 The second can be worked around with Pulsar REST/MQ (with a performance/throughput penalty)
 
+???
+
+- For the DRMs which require a shared filesystem there are additional requirements.
+- First, Galaxy and the tool dependencies are at the same location on the head and compute nodes.
+- Job directories must be in a shared location on both head and compute nodes.
```
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/admin/tutorials/connect-to-compute-cluster/slides.html:263]: yeah, good point. I thought we moved that somewhere, but, I guess not. I've added an extra tip box in the tutorial.

**Comment by hexylena** [comment]
Fixed @natefoo :)

**Review by shiltemann — APPROVED** [review]

**Pull request merged**